conf/ansible-inventory

```diff
@@ -1,32 +0,0 @@
-[OSEv3:children]
```

Comment:
do we already store these inside oVirt?

Reply:
yes. I copy it to the metrics store machine.